vayesta/ewf/fragment.py

```diff
@@ -280,8 +280,9 @@ def get_solver_options(self, solver):
             self.log.debugv("Passing fragment option %s to solver.", attr)
             solver_opts[attr] = getattr(self.opts, attr)
 
-        if solver.upper() == 'TCCSD':
-            solver_opts['tcc'] = True
+        has_actspace = ((solver == "TCCSD") or ("CCSDt" in solver) or
+                        (solver == "EBCC" and self.opts.solver_options['ansatz'] == "CCSDt"))
```

Comment:
Can we maybe do `solver.upper() == "EBCC"`? I stylise `ebcc` as lower-case

Reply:
as in, I stylise the code that way, but the actual solver classes in `ebcc` are indeed caps, so I guess maybe this makes sense

Reply:
It depends- I was previously just using `solver.upper()` in the solver selection which would have done this automatically, but to support CCSDt' as an option I changed that in this PR. I was wondering about your prefered stylising (I plumped for `ebCC` above)- I could just change to `solver.upper()`for this specific check?

Reply:
I've also just realised this is checking `=="CCSDt"` not `=="CCSDt'"`, which I'll fix at the same time. That prime is a little awkward, but I'm assuming CCSDt is a separate thing? It's been a while since I checked the literature for this.

Reply:
That's fair, I don't think I've used anywhere else and have updated the above to reflect that.
I've updated this so that capitalisation of `eb` is arbitrary, along with `cc` if no ansatz is specified in the solver. To be compatible with `ebcc`'s ansatz specification we need the ansatz to be case-sensitive, so for instance we'd allow `ebcc`, `ebCCSD`, or `EBCCSD` but not `ebccSD`. Does that sound like a reasonable compromise?

Reply:
sure, any case can work really if we just throw in an `upper`

Reply:
Given `ebcc` requires case-sensitivity in ansatz specification I think requiring it for Vayesta is acceptable- just so that we don't end up with unexpected interactions down the line eg. someone doesn't think they're running a CCSDt calculation down the line and ending up with an untenable CCSDT? Basically, given that capitalisation has meaning in CC ansatzes we probably can't just always use `solver.upper()` without a bit more documentation of supported approaches.

Reply:
so I mean for `EBCC` specifically, like `ebcc == ebCC == EBCC`, but yea if you're specifying a specific CC ansatz then case sensitivity is important.

Reply:
Sweet, yeah that's what we've got